account_avatax/models/avatax_rest_api.py

```diff
@@ -319,6 +319,16 @@ def get_tax(
 
         response = self.client.create_or_adjust_transaction(data)
         result = self.get_result(response, ignore_error=ignore_error)
+        self.config.env["avatax.log"].sudo().create(
```

Comment:
This is a good feature - we needed a log on the Odoo side.
But I would only do this if the logging is enabled or there are notification users configured.